src/main/scala/org/scalacheck/ScalaCheckFramework.scala

```diff
@@ -218,13 +222,13 @@ final class ScalaCheckFramework extends Framework {
 
   }
 
-  def slaveRunner(_args: Array[String], _remoteArgs: Array[String],
+  def slaveRunner(_args: Array[String | Null], _remoteArgs: Array[String | Null],
```

Comment:
A Scala method overrides some Java method. We need to add |Null to the method's parameter types to make the override still work.